### DISCUSSION TITLE
pool stats - update interval?

### DISCUSSION THREAD

**Discussion by MiningCryptoLive** [coinfoundry/miningcore#1645] [Q&A]
Is there a setting that can be changed to update the pool stats more frequently? If so, How?
Thanks

**Comment by edcomrocha** [comment]
I couldn't test yet.
But couldn't it be here?

https://github.com/oliverw/miningcore/blob/master/src/Miningcore/config.schema.json#L802

**Comment by edcomrocha** [comment]
I tested it, pool stats are updated here.

One thing I'm still looking for is updating the % confirmation of blocks.
It seems that it is only updated with each payment, but I still couldn't be sure of that, because I couldn't find where they are updated


`"statistics": {`
`"updateInterval": 15, //default 120`
`"hashrateCalculationWindow": 10,`
`"gcInterval": 1,`
`"cleanupDays": 720`
`},`

**Comment by MiningCryptoLive** [comment]
@edcomrocha Thanks for this. Works perfectly. What about the confirmation % are you looking to change? How many Block confirmations before confirmed?

**Reply by edcomrocha** [reply]
Actually I want confirmationProgress": to be updated in a shorter amount of time.

What I have noticed is that in my API, it only updates during payments.
If I have a currency that pays only every 6 hours, it will only show changes in the API after that payment period.

![image](https://user-images.githubusercontent.com/6929674/226219619-f8a3ce2c-4fac-4d12-a697-6590f8085ed5.png)

**Reply by ghost** [reply]
Yes that's true. It only updates the confirmation status when payout cycle. I'm looking for a solution to this as well.

**Reply by edcomrocha** [reply]
If you find something please let us know.
I've been looking for a while and haven't found it.

@oliverw , the verification of blocks only occurs when payments occur, or is there any config that I can change for this verification if I don't have what I want.

Sorry if I'm missing something, but I just don't think so.
